blueprints.yaml

```diff
@@ -125,6 +125,30 @@ form:
           0: PLUGIN_ADMIN.NO
       validate:
           type: bool
+
+    Background:
+      type: section
+      title: Background execution
+      underline: true
+    execute_in_background.enabled:
+      type: toggle
+      label: Execute in background
+      help: Executes the synchronization process in the background, improving the perceived performance of the Saving process
+      highlight: 1
+      default: 0
+      options:
+        1: Yes
+        0: No
+      validate:
+        type: bool
+
+    execute_in_background.root_path:
+      type: text
+      label: Absolute path to Grav's root folder
+      description: >
+        Only fill this field if either /bin or /user is not installed in Grav's root folder.<br/>
+        You may need to hit the Save button twice the first time you set this path.
```

Comment:
`If you enabled the git sync trigger on all admin save event,` you may need to hit the Save button twice the first time you set this path.